frontend/src/pages/LogIn/index.tsx

```diff
@@ -77,38 +61,53 @@ const LogIn: React.FC = () => {
           <Stack
             component="form"
             direction="column"
-            spacing={1.5}
             sx={(theme) => ({
               marginTop: theme.spacing(2),
               marginBottom: theme.spacing(2),
             })}
-            onSubmit={handleLogIn}
+            onSubmit={handleSubmit((data) => login(data.email, data.password))}
             noValidate
           >
-            <CustomTextField
+            <TextField
               label="Email"
-              size="small"
               required
-              emptyField={emptyFields.email}
-              validator={emailValidator}
-              onChange={(value, isValid) =>
-                handleInputChange("email", value, isValid)
-              }
+              fullWidth
+              margin="normal"
+              type="email"
+              {...register("email", { validate: { emailValidator } })}
+              error={!!errors.email}
+              helperText={errors.email?.message}
             />
-            <CustomTextField
+            <TextField
               label="Password"
-              size="small"
               required
-              emptyField={emptyFields.password}
-              onChange={(value, isValid) =>
-                handleInputChange("password", value, isValid)
-              }
-              isPasswordField
+              fullWidth
+              margin="normal"
+              {...register("password", { validate: { passwordValidator } })}
```

Comment:
As compared to the sign up page where we are making sure the user creates a strong password, I think on this page we shouldn't be giving them error messages about the password requirements? Would it be more appropriate to just call the login API which would then indicate that the given password is invalid because of a mismatch?

To cater for the scenario of the user leaving the field blank, maybe we could do this instead:
`{...register("password", { required: "Password is invalid" })}`